### PR TITLE
Feature/sad update

### PR DESCRIPTION
**Pull request by Razumain**
Updates to the technical framework to support the new CEN standards for remote signing with regard to Sole Control Assurance Level 2 (SCAL2).

### CODE REVIEW

File: schemas/EidCsigSAP.xsd [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (martin-lindstrom) [inline schemas/EidCsigSAP.xsd]: An XML schema using the xs:documentation element makes me happy.

File: schemas/EidCentralSigDssExt-1.1.xsd [hunk 2]
```diff
@@ -0,0 +1,370 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (martin-lindstrom) [inline schemas/EidCentralSigDssExt-1.1.xsd]: OK, great. I suggest that we publish our schemas at the same place where we publish the specs, under http://elegnamnden.github.io. 

If we are to publish them under, say http://elegnamnden.github.io/schemas, we would have to set upp a new repo named "schemas". Maybe that is a good idea.

File: ELN-0613 - Signature Activation Protocol.md [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+`<Eid2Element>`, `<ns:ForeignElement>`, `Attribute`, **Datatype**,
+`OtherCode`.
+
+<a name="signature-activation-protocol"></a>
+## 2. Signature Activation Protocol
+
+<a name="scope"></a>
+### 2.1. Scope
+The scope of the Signature Activation Protocol (SAP) is to support request for and delivery of the Signature Activation Data (SAD) to the Signature Activation Module (SAM). The SAM is a tamper resistant module inside the Remote Signing Service which validates the SAM in order to ensure that:
+
```
Comment (martin-lindstrom) [inline ELN-0613 - Signature Activation Protocol.md]: "validates the SAM in order to ...". Det ska väl vara "validates the SAD in order to ..."?

File: ELN-0613 - Signature Activation Protocol.md [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+- The signer is properly authenticated.
+- The signer agrees to sign the data to be signed.
+- The correct signing key for this signer and this instance of signing is properly identified.
+
+The federated signing model does not use pre-assigned signing keys. Instead, a new signing key is generated for each sign request then permanently deleted. This particular use-case is recognised by prEN 419 241-1 [[RSIG-PP-1](#rsig-pp-1)] and prEN 419 241-2 [[RSIG-PP-2](#rsig-pp-2)], which under these conditions allows the signature key reference to be implicit and derived from the signer's identity. For the present implementation of the SAP the following data is included in the SAD:
+
+- The signer's identity
+- Information about how the signer was authenticated and by who
```
Comment (martin-lindstrom) [inline ELN-0613 - Signature Activation Protocol.md]: by whom

File: ELN-0602 - Deployment Profile for the Swedish eID Framework.md [hunk 5]
```diff
@@ -1110,6 +1112,31 @@ associated with requests from signature services:
     authentication context which includes display of a sign message,
     even if the request has no present `ForceAuthn` attribute or includes
     a `ForceAuthn` attribute set to the value `false`.
+    
+<a name="requesting-scal2-signature-activation-data"></a>
+#### 7.2.2. Requesting SCAL2 Signature Activation Data
+
+The type of signature requested by a signature request is, according to [EidDSS_Profile], specified by the `CertType` attribute of the `<CertRequestProperties>` element. When the value of this attribute is set to `QC/SSCD`, the requested signature is a Qualified Signature created in a Qualified Signature Creation Device (QSCD). To achieve this level of signature the Authentication Request MUST include a request for Signature Activation Data (SAD) for Sole Control Assurance Level 2 (SCAL2) in accordance with the "Signature Activation Protocol for Federated Signing" [SigSAP].
+
+A SAD returned from the Identity Provider MUST have a signature which can be verified by a Certificate in the Identity Provider's metadata. The signature algorithm used to sign the SAD MUST be equivalent to the algorithm used to sign the responses and assertions from the Identity Provider.
```
[the remark this answers is not in the document]
Comment (Razumain) [reply ELN-0602 - Deployment Profile for the Swedish eID Framework.md]: Accepted

File: ELN-0602 - Deployment Profile for the Swedish eID Framework.md [hunk 6]
```diff
@@ -1110,6 +1112,31 @@ associated with requests from signature services:
...
+
+Identity Providers SHALL advertise support for the SAP protocol according to [SigSAP], by including the service property entity category URI
```
[the remark this answers is not in the document]
Comment (Razumain) [reply ELN-0602 - Deployment Profile for the Swedish eID Framework.md]: The second bullet and example is moved

File: ELN-0606 - Entity Categories for the Swedish eID Framework.md [hunk 7]
```diff
@@ -338,6 +340,18 @@ authentication using mobile devices.
 See \[EidDiscovery\] for a more extensive explanation of the use of the
 mobile-auth category.
 
+<a name="scal2"></a>
+### 3.2. SCAL2
```
[the remark this answers is not in the document]
Comment (Razumain) [reply ELN-0606 - Entity Categories for the Swedish eID Framework.md]: OK

File: ELN-0613 - Signature Activation Protocol.md (hunk 3 above)
Comment (Razumain) [reply ELN-0613 - Signature Activation Protocol.md]: Rättat

File: ELN-0613 - Signature Activation Protocol.md (hunk 4 above)
Comment (Razumain) [reply ELN-0613 - Signature Activation Protocol.md]: Rättat

**Pull request merged**